project/dexter-peer-review/src/java/com/samsung/sec/dexter/peerreview/plugin/DexterPeerReviewPlugin.java

```diff
@@ -0,0 +1,263 @@
+		ArrayList<DPRComment> majDefects = Lists.newArrayList();
+		ArrayList<DPRComment> crcDefects = Lists.newArrayList();
+
+		for (DPRComment comment : comments) {
+			if ("CRI".contains(comment.getSeverity())) {
+				criDefects.add(comment);
+			} else if ("MAJ".contains(comment.getSeverity())) {
+				majDefects.add(comment);
+			} else {
+				crcDefects.add(comment);
+			}
+		}
+
+		List<Defect> defectList = new ArrayList<Defect>();
+		if (!criDefects.isEmpty()) {
+			defectList.add(makeDefect(config, "CRI", criDefects));
+		}
+		if (!majDefects.isEmpty()) {
+			defectList.add(makeDefect(config, "MAJ", majDefects));
+		}
+		if (!crcDefects.isEmpty()) {
+			defectList.add(makeDefect(config, "CRC", crcDefects));
+		}
+
+		return defectList;
+	}
+
+	private Defect makeDefect(AnalysisConfig config, String severity, ArrayList<DPRComment> defectInfos) {
+		Defect defect = new Defect();
+
+		defect.setCheckerCode("DPR_" + severity);
+		defect.setFileName(config.getFileName());
+		defect.setModulePath(config.getModulePath());
+		defect.setClassName("");
+		defect.setMethodName("");
+		defect.setLanguage(config.getLanguageEnum().toString());
+		defect.setSeverityCode(severity);
+		defect.setCategoryName("PeerReview");
+		defect.setMessage("");
+		defect.setToolName(PLUGIN_NAME);
+		defect.setAnalysisType(config.getAnalysisType().name());
+
+		for (DPRComment defectInfo : defectInfos) {
+			Occurence occ = new Occurence();
+			occ.setStartLine(defectInfo.getStartLine());
+			occ.setEndLine(defectInfo.getEndLine());
+			occ.setStringValue("DPR");
+			occ.setMessage(defectInfo.getReviewComment());
+
+			defect.addOccurence(occ);
+		}
+
+		return defect;
+	}
+
+	static String getFullCommentWithoutLineSeparator(String comment) {
+		return comment.replace("\r\n", "").replace("\n", "").trim();
+	}
+
+	private HashMap<String, String> getSeverityAndCommentFromFullComment(String comment) {
+		String severity = "";
+		String reviewComment = "";
+
+		Pattern criticalPattern = Pattern.compile(criticalRegExp);
+		Matcher criticalMatcher = criticalPattern.matcher(comment);
+
+		Pattern majorPattern = Pattern.compile(majorRegExp);
+		Matcher majorMatcher = majorPattern.matcher(comment);
+
+		Pattern defaultPattern = Pattern.compile(simpleRegExp);
+		Matcher defaultMatcher = defaultPattern.matcher(comment);
+
+		if (criticalMatcher.find()) {
+			severity = "CRI";
+			reviewComment = comment.substring(criticalMatcher.end());
+		} else if (majorMatcher.find()) {
+			severity = "MAJ";
+			reviewComment = comment.substring(majorMatcher.end());
+		} else if (defaultMatcher.find()) {
+			severity = "CRC";
+			reviewComment = comment.substring(defaultMatcher.end());
+		}
+
+		HashMap<String, String> infoHashMap = new HashMap<String, String>();
+		infoHashMap.put("severity", severity);
+		infoHashMap.put("comment", reviewComment.trim());
+
+		return infoHashMap;
+	}
+
+	static int getLineFromSourcecode(CharSequence sourcecode, int start) {
```

Comment:
The name of second parameter 'start' makes the function difficult to understand.
The signature of matcher.region() is ` Matcher region(int start, int end)`
The variable 'start' is passed to the parameter 'int end'.